pctPolynomialMLPFunction.txx

```diff
@@ -233,52 +118,6 @@ PolynomialMLPFunction
   }
   m_dm_y[m_PolynomialDegree+3] = m_c_y[1] * m_bm[m_PolynomialDegree] / (m_PolynomialDegree+2) / (m_PolynomialDegree+3);
 
-// // For testing stuff only
-// itk::Vector<double, 9> test = m_dm_y[0] * m_dm_y;
-// for(int i = 0; i <= m_PolynomialDegree+3; i++)
-// {
-//   // std::cout << i << ": d_x = " << m_dm_x[i] << std::endl;
-//   std::cout << i << ": d_y = " << m_dm_y[i] << std::endl;
-//   std::cout << i << ": test = " << test[i] << std::endl;
-// }
-// ****
-
-}
-
-void
-PolynomialMLPFunction
-::Evaluate( const double u, double &x, double&y, double &dx, double&dy )
-{
-#ifdef MLP_TIMING
-  m_EvaluateProbe1.Start();
-#endif
-  const double u1 = u-m_uOrigin;
-
-
-
-// #ifdef MLP_TIMING
-//   m_EvaluateProbe1.Stop();
-//   m_EvaluateProbe2.Start();
-// #endif
-
-  x = 0;
-  y = 0;
-
-  for(int i = 0; i != m_PolynomialDegreePlusThree; i++)
-  {
-    x += m_dm_x[m_PolynomialDegreePlusThree-i];
-    x *= u1;
-    y += m_dm_y[m_PolynomialDegreePlusThree-i];
-    y *= u1;
-  }
-
-  x += m_dm_x[0];
-  y += m_dm_y[0];
-
-
-#ifdef MLP_TIMING
-  m_EvaluateProbe1.Stop();
-#endif
 }
```

Comment:
Why was this deleted? Intentional?

Reply:
That was a version of the function which I had implemented in the beginning. It calculates the MLP at a single depth and not a whole array of depth. The vectored version of the function is much faster. The DistanceDrivenProjection only uses the vectored version so I saw no motivation to keep the old one.